cli/ops/dispatch_json.rs

```diff
@@ -37,7 +37,19 @@ fn serialize_result(
     Ok(v) => json!({ "ok": v, "promiseId": promise_id }),
     Err(err) => json!({ "err": json_err(err), "promiseId": promise_id }),
   };
-  let vec = serde_json::to_vec(&value).unwrap();
+  let mut vec = serde_json::to_vec(&value).unwrap();
+
+  debug!("JSON response pre-align, len={}", vec.len());
+  let len = vec.len();
+  let modulo = len % 4;
```

Comment:
This works, no comment about that.
That said, this whole section can be replaced by a one-liner:
```rust
vec.resize((vec.len() + 3usize) & !4usize, b' ');
```

Reply:
Please leave a comment with that--^